content/tutorials/wallets.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Wallets for Ocean Tokens
+description: Learn about wallets that can hold Ocean Tokens.
+---
+
+If you want to use the Ocean Network, you need Ocean Tokens, and to get Ocean Tokens, you need to get a wallet that can hold Ocean Tokens.
```

Comment:
this should be the description above. And please mention right here that it's an ERC-20 token cause nobody outside of Ocean knows that cause for whatever reason marketing does not deem it necessary to communicate that. For the outside world it's a "native utility token" and no ERC-20 token wallet is supported. This article now says the complete opposite so that info should be right up here